src/test/java/com/wikia/webdriver/testcases/mobilewikitests/NavigationMobileWikiTests.java

```diff
@@ -2,40 +2,47 @@
 
 import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
 import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.annotations.InBrowser;
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.elements.mercury.components.GlobalNavigationMobile;
 import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
-
 import org.testng.annotations.Test;
 
 @Test(groups = "MobileWiki_Navigation")
 @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
 @Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
-public class NavigationMobileWikiTests extends NavigationTests {
+public class NavigationMobileWikiTests {
 
   //test disabled until we find solution how to click 'Wikis' link in navigation
   @Test(groups = "mobileWiki_navigation_openAndCloseNavigationAndItsSubMenu", enabled = false)
   public void mobileWiki_navigation_openAndCloseNavigationAndItsSubMenu() {
-    super.mobileWiki_navigation_openAndCloseNavigationAndItsSubMenu(
-        new ArticlePage().open(MercurySubpages.MAIN_PAGE)
-    );
+    GlobalNavigationMobile nav = new ArticlePage().open(MercurySubpages.MAIN_PAGE).getGlobalNavigationMobile().openNavigation();
+    Assertion.assertTrue(nav.isFirstLevelMenuVisible());
+
+    nav.clickCloseButton();
+    Assertion.assertFalse(nav.isFirstLevelMenuVisible());
   }
 
   @Test(groups = "mobileWiki_navigation_navigationOnEnglishWiki")
   public void mobileWiki_navigation_navigationOnEnglishWiki() {
-    super.mobileWiki_navigation_navigationOnEnglishWiki(
-        new ArticlePage().open(MercurySubpages.MAIN_PAGE)
-    );
+    GlobalNavigationMobile nav = new ArticlePage().open(MercurySubpages.MAIN_PAGE).getGlobalNavigationMobile().openNavigation();
+    Assertion.assertTrue(nav.isFirstLevelMenuVisible());
+
+    nav.clickCloseButton();
+    Assertion.assertFalse(nav.isFirstLevelMenuVisible());
   }
 
   @Execute(onWikia = MercuryWikis.DE_WIKI, language = "de")
   @Test(groups = "mobileWiki_navigation_navigationOnNonEnglishWiki")
   public void mobileWiki_navigation_navigationOnNonEnglishWiki() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00100)